lte/gateway/c/core/oai/lib/gtpv2-c/gtpv2c_ie_formatter/src/gtpv2c_ie_formatter.c

```diff
@@ -361,13 +361,22 @@ nw_rc_t gtpv2c_ambr_ie_set(nw_gtpv2c_msg_handle_t* msg, ambr_t* ambr) {
   p_ambr = ambr_br;
 
   memset(ambr_br, 0, 8);
+  // if for bps or kbps
 
-  INT32_TO_BUFFER((ambr->br_ul / 1000), p_ambr);
-  p_ambr += 4;
+  if (ambr->br_unit == BPS) {
+    INT32_TO_BUFFER((ambr->br_ul / 1000), p_ambr);
+    p_ambr += 4;
 
-  INT32_TO_BUFFER(((ambr->br_dl / 1000)), p_ambr);
-  // todo: byte order?
+    INT32_TO_BUFFER(((ambr->br_dl / 1000)), p_ambr);
+    // todo: byte order?
+  }
+
+  else {
```

Comment:
this is a legit warning. please run clang formatter (running `make run` or `make build_oai` should format this correctly.

Reply:
check these in switch statements and explicitly define cases for each supported units. Provide a default error message, handling..